app/javascript/admin/controllers/creator_settings_controller.js

```diff
@@ -2,78 +2,11 @@ import { Controller } from '@hotwired/stimulus';
 import { isLowContrast } from '@utilities/color/contrastValidator';
 import { brightness } from '@utilities/color/accentCalculator';
 
-const MAX_LOGO_PREVIEW_HEIGHT = 80;
```

Comment:
I pulled out the logo related stuff and put it into it's own controller so it can be used elsewhere.